addons/surgery/functions/fnc_reboaDeepApplyLocal.sqf

```diff
@@ -0,0 +1,36 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Mazinski
+ * Local call for applying Deep REBOA.
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Body Part <STRING>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [player, cursorObject, "LeftLeg"] call kat_surgery_fnc_reboaDeepApplyLocal
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient", "_bodyPart"];
+
+private _partIndex = ALL_BODY_PARTS find toLower _bodyPart;
```

Comment:
its not used in this file, i guess it should be used down below instead of 4/5numbers?